pkg/bpf/bpf_linux.go

```diff
@@ -113,7 +113,16 @@ func UpdateElementFromPointers(fd int, structPtr unsafe.Pointer, sizeOfStruct ui
 	}
 
 	if ret != 0 || err != 0 {
-		return fmt.Errorf("Unable to update element for map with file descriptor %d: %s", fd, err)
+		switch err {
+		case unix.E2BIG:
+			return fmt.Errorf("Unable to update element for bpf map with file descriptor %d: the map is full, please consider resizing it. %w", fd, err)
```

Comment:
I'm wondering whether it's possible in some cases to provide more useful info than FD, e.g. a map name. Have you checked how difficult is to pass this information down to the function?

Reply:
It is possible. The previous commit does it. Should we do it?

Reply:
Was there any objection from other reviewers regarding adding this info?

Reply:
No as far as I know.